website/src/components/Dictionary/index.tsx

```diff
@@ -5,11 +5,12 @@ import Schema from '../Schema';
 
 const Dictionary = ({ schemas, menuContents, isLatestSchema, isDiffShowing }) =>
   schemas.length > 0 ? (
-    schemas.map((schema) => {
+    schemas.map((schema, i) => {
       const menuItem = find(menuContents, { name: startCase(schema.name) });
 
       return (
         <Schema
+          key={i}
```

Comment:
using `i` as `key` is fine if the following are true:
1) there's no unique identifier for the `schema`
2) the order of the list is not expected to change.

If there is a unique identifier, it'd be much preferable to use that instead of `i`, because it's hard to tell from the code whether the order never changes. Let's consider this throughout :)